tests/redux/reducers/account/accountReducer-test.js

```diff
@@ -191,11 +192,11 @@ describe('accountReducer', () => {
         const mockActionState = {
             objectClass: mockOCAPI.results,
             objectClassDefinitions: {
-                '1': 'First Major',
-                '2': 'First Major Minor'
+                1: 'First Major',
```

Comment:
Did these change from strings to numbers?

Reply:
I did not make those changes, i think that came from my linter autosave making corrections.